core/lib/zksync_core/src/l1_gas_price/gas_adjuster/mod.rs

```diff
@@ -130,8 +183,82 @@ impl GasAdjuster {
     }
 
     pub(crate) fn estimate_effective_pubdata_price(&self) -> u64 {
-        // For now, pubdata is only sent via calldata, so its price is pegged to the L1 gas price.
-        self.estimate_effective_gas_price() * L1_GAS_PER_PUBDATA_BYTE as u64
+        // TODO: remove constant and use setting from config instead.
```

Comment:
Waiting for https://github.com/matter-labs/zksync-era/pull/1243 to avoid conflicts